charts/akeyless-api-gateway/templates/deployment.yaml

```diff
@@ -20,6 +20,11 @@ spec:
       labels:
         {{- include "akeyless-api-gw.selectorLabels" . | nindent 8 }}
     spec:
+      {{- if .Values.deployment.securityContext.enabled }}
+      securityContext:
+         runAsUser: {{ .Values.deployment.securityContext.runAsUser }}
+         fsGroup: {{ .Values.deployment.securityContext.fsGroup }}
+      {{- end }}
```

Comment:
What are those uid and gid?

Reply:
Here the uid is comming from the values file